cou/apps/auxiliary.py

```diff
@@ -196,35 +196,8 @@ def pre_upgrade_steps(
         :rtype: list[PreUpgradeStep]
         """
         steps = super().pre_upgrade_steps(target, units)
-        # Since auto restart is disabled, we don't know the if the service
-        # has pending events or not, so we want to run `run-deferred-hooks`
-        # action to clear the events before performing upgrade. If there
-        # are no pending events, this step should be a no-op, so it's safe
-        # to run anyway
         if self.config.get("enable-auto-restarts", {}).get("value") is False:
-            # Run any deferred events and restart the service. See
-            # https://charmhub.io/rabbitmq-server/actions#run-deferred-hooks
-            units_to_run_action = self.units.values() if units is None else units
```

Comment:
Old logic runs on the provided `units` arg if available.  The new logic ignores this arg.  I'm not sure if this is a problem?

Reply:
I don't think the `units` is even used. I check the root where the `units` is passed, and it kept as the default, which is `None` https://github.com/canonical/charmed-openstack-upgrader/blob/911328c1354340b8c59819c6b46519cdb14e41cf/cou/steps/plan.py#L758

If you prefer to be consistent, I can added that back

Reply:
yeah that makes sense.  I don't mind; we can refactor that units arg out later. :)

Reply:
Okay, let me add that back, and refactor the use of `units` in the future if it's necessary

Reply:
Done